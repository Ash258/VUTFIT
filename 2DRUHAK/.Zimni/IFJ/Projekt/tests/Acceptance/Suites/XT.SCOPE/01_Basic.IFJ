' Basic scopes

scope
	dim a as integer = 42
	scope
		print a;
		dim a as integer = 43
		print a;
	end scope
	print a;
	scope
		print a;
		dim a as integer = 43
		print a;
	end scope
	print a;
end scope
