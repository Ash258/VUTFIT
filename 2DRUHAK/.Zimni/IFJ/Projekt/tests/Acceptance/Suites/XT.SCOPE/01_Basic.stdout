 42 43 42 42 43 42